src/main/java/uk/ac/sanger/sccp/stan/service/operation/confirm/ConfirmSectionServiceImp.java

```diff
@@ -179,6 +186,14 @@ public ConfirmLabwareResult confirmLabware(User user, ConfirmSectionLabware csl,
                 measurements.add(new Measurement(null, "Thickness", String.valueOf(pa.getSampleThickness()),
                         sample.getId(), null, slot.getId()));
             }
+            if (!nullOrEmpty(sec.getRegion())) {
+                samplePositions.add(new SamplePosition(slot.getId(), sample.getId(), slotRegions.get(sec.getRegion()), null));
+            }
```

Comment:
Do we need to check whether slot have multiple sections here, as we need to add regions only for those?

Reply:
It doesn't seem necessary to me: I'm happy that if the request specifies a region, we record it.

Reply:
Ok